Template/Pages/Long-term Staff.html

```diff
@@ -0,0 +1,10 @@
+<!--
+ Title: [* Προσωπικό */* Long-term Staff *]
+ -->
+[*
+<p>...</p>
+*/*
+<p> We are looking for people who: share the same values, will commit to Greek language learning, are flexible and adaptable, are self motivated, take initiative, persevere, can handle the different challenges that come with a pioneering situation, don’t compromise easily. </p>
+<p><b> Long-term commitment (2 years*) requires: </b></p>
```

Comment:
[`<strong>`](https://www.w3schools.com/tags/tag_strong.asp) is meaningful to things like screen readers. `<b>` is deprecated and only affects the appearance.

[`<sup>`](https://www.w3schools.com/tags/tag_sup.asp) defines superscripts. (Depending on the fonts on the user’s device, the plain asterisk won’t necessarily appear superscripted by default. This might happen: `2 years*`)
```suggestion
<p><strong> Long‐term commitment (2 years<sup>*</sup>) requires: </strong>
```